mrtt-ui/src/views/Sites.js

```diff
@@ -1,8 +1,48 @@
-import React from 'react'
+import { Link } from 'react-router-dom'
+import { Stack } from '@mui/material'
+import { toast } from 'react-toastify'
+import axios from 'axios'
+import LoadingIndicator from '../components/LoadingIndicator'
+import React, { useEffect, useState } from 'react'
 
-/** Scaffold component for the /sites route */
+import { ButtonPrimary } from '../styles/buttons'
+import { H4 } from '../styles/typography'
+import { LinkCard, PagePadding, RowSpaceBetween } from '../styles/containers'
+import language from '../language'
+const sitesUrl = `${process.env.REACT_APP_API_URL}/sites/`
 function Sites() {
-  return <div>Sites (views/Sites)</div>
+  const [isLoading, setIsLoading] = useState(true)
+  const [sites, setSites] = useState([])
+
+  useEffect(function loadSitesData() {
+    axios
+      .get(sitesUrl)
+      .then(({ data }) => {
+        setIsLoading(false)
+        setSites(data)
+      })
+      .catch(() => toast.error(language.error.apiLoad))
+  }, [])
+
+  const sitesList = sites.map(({ site_name, id }) => (
```

Comment:
nice!  I see you are getting the data for each site here which is great - does this mean we do not need an initial fetch any more for the other form sections?